.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,52 @@
+## Description
+
+*Replace this paragraph with a description of what this PR is doing. If you're
+modifying existing behavior, describe the existing behavior, how this PR is
+changing it, and what motivated the change.
+
+## Related Issues
+
+*Replace this paragraph with a list of issues related to this PR from our [issue
+database]. Indicate, which of these issues are resolved or fixed by this PR.
+There should be at least one issue listed here.*
+
+## Tests
+
+I added the following tests:
+
+*Replace this with a list of the tests that you added as part of this PR. A
+change in behaviour with no test covering it will likely get reverted
+accidentally sooner or later. PRs must include tests for all
+changed/updated/fixed behaviors. See [testing the engine] for instructions on
+writing and running engine tests.*
+
+## Checklist
+
+Before you create this PR confirm that it meets all requirements listed below by checking the relevant checkboxes (`[x]`). This will ensure a smooth and quick review process.
+
+- [ ] I read the [Contributor Guide] and followed the process outlined there for submitting PRs.
+- [ ] I signed the [CLA].
+- [ ] I read and followed the [C++, Objective-C, Java style guides] for the engine.
+- [ ] I read the [Tree Hygiene] wiki page, which explains my responsibilities.
+- [ ] I updated/added relevant documentation.
+- [ ] All existing and new tests are passing.
+- [ ] I am willing to follow-up on review comments in a timely manner.
+
+## Breaking Change
+
+Did any tests fail when you ran them? Please read [Handling breaking changes].
+
+- [ ] No, no existing tests failed, so this is *not* a breaking change.
+- [ ] Yes, this is a breaking change. *If not, delete the remainder of this section.*
+   - [ ] I wrote a design doc: https://flutter.dev/go/template *Replace this with a link to your design doc's short link*
+   - [ ] I got input from the developer relations team, specifically from: *Replace with the names of who gave advice*
+   - [ ] I wrote a migration guide: https://flutter.dev/go/breaking-changes-template *Replace this with a link to a pull request that adds the migration guide to https://flutter.dev/docs/release/breaking-changes*
```

Comment:
This guidance has seldom proven applicable for engine changes. How about just mentioning changes to the public interfaces in the embeddings?

Reply:
I agree it's less common. Public embedding API would be breaking changes. But I'm open to removing this section if you think it's rare enough for the engine repo.

Reply:
Don't have a strong opinion either way. Lets land this and tweak as necessary.